torch/fx/experimental/proxy_tensor.py

```diff
@@ -805,7 +836,7 @@ def wrap_fake(x):
         # purpose of `make_fx` is to produce graphmodules as a side effect; its internal execution is
         # thus irrelevant to any external functional trace.
         with decompose(decomposition_table), fake_tensor_mode, python_dispatcher_mode, pre_dispatch_mode, proxy_function_mode, \
-             sym_mode, proxy_mode, disable_autocast_cache(), disable_proxy_modes_tracing(enable_current=True):
+             sym_mode, proxy_mode, disable_autocast_cache():
```

Comment:
Tell me more?

Reply:
`disable_proxy_modes_tracing(enable_current=True)` would originally:
* Look at **all** of the proxy modes on the stack
* disable all of the except for the most recent one

Now that we can only have one proxy mode active at a time, this is a no-op: the new `ProxyTorchDispatchMode.__enter__` logic basically does this implicitly by temporarily disabling any previous proxy mode.